systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -2194,8 +2194,8 @@ void testLoggingHierarchy() {
         }
     )
     void testChangingInternalToExternalLoggingTriggerRollingUpdate() {
-        // Cruise Control currently does not work with Kafka 4.0 and this tests is therefore disabled when Kafka 4.0 or newer is used.
-        // This should be re-enabled once Cruise Control support is fixed: https://github.com/strimzi/strimzi-kafka-operator/issues/11199
+        // This test currently does not work with Kafka 4.0 due to some logging related changes and is therefore disabled when Kafka 4.0 or newer is used.
+        // This should be re-enabled once we fix the issue: https://github.com/strimzi/strimzi-kafka-operator/issues/11312
```

Comment:
This is wrong.
Because if with the log4j2 the config is dynamically changed, I would update the comment that "it should be deleted once Kafka 3.9.x is dropped from support".

Reply:
Isn't think thing still something you want to test? Just the test would be different ...

Reply:
We discussed it offline with @ShubhamRwt and based on his observation, it will roll anyway even with the different configuration. That's why I suggested that we can possibly remove the test, because we are testing the dynamic configuration somewhere else. 

Or do you mean different like - check that when we do this change, there will be no rolling update?

Reply:
Yeah, that was my understanding as we are testing dynamic logging in some places and we are expecting no rolling since with Kafka 4.0.0 we get dynamic logging with broker etc too.

Reply:
If you say that it `will roll anyway` the test still makes sense? (but not sure why would it roll) But even without the rolling, transition between external and internal logging and vice versa might be something to test?

Reply:
I assume Lukas meant not roll since we are on Kafka 4.0.0

Reply:
But I get your point, I can fix the test for the same. I think then we would assume a roll if Kafka is 3.9.0 and if its 4.0.0 then we don't expect a roll but check if the logging is updated

Reply:
Separate PR is fine if this is not related to CC's Kafka 4.0 support. I question more the idea to remove it rather than that it needs to be done in this PR.

Reply:
Yeah good point.
So I guess (once again) we should maybe change the issue description a bit and change it in different PR, so we don't block this one.

Reply:
I have updated the comment and added the issue tracking this and will create another PR to fix it.

Reply:
Thanks :) btw you don't need to do it, the original idea was that if it would be just change of comment, we don't need to have another issue and we can do it here.